docs/management/managing-remote-clusters.asciidoc

```diff
@@ -1,26 +1,64 @@
-[[managing-remote-clusters]]
-== Managing Remote Clusters
+[[working-remote-clusters]]
+== Working with remote clusters
+
+{kib} *Management* provides user interfaces for working with data from remote 
+clusters and managing the cross cluster replication process. Replicating indices from a 
+leader remote cluster to a follower index in a local cluster can be used to provide 
+remote backups for disaster recovery or for geo-proximite copies of data.
+
+Before using these features, you should be familiar with the following concepts:
 
-{kib} *Management* provides two user interfaces for working with data from remote 
-clusters. 
+* {xpack-ref}/xpack-ccr.html[Cross cluster replication]
+* {ref}/modules-cross-cluster-search.html[Cross cluster search]
+* {xpack-ref}/cross-cluster-configuring.html[Cross cluster security requirements]
+
+[float]
+[[managing-remote-clusters]]
+== Managing remote clusters 
 
 *Remote Clusters* helps you manage remote clusters for use with 
-{ref}/modules-cross-cluster-search.html[cross cluster search] and 
-{xpack-ref}/xpack-ccr.html[cross cluster replication]. You can add and remove remote 
+cross cluster search and cross cluster replication. You can add and remove remote 
 clusters and check their connectivity. 
 
+Before you use this feature, you should be familiar with the concept of 
+{ref}/modules-remote-clusters.html[remote clusters].  
+ 
 Go to *Management > Elasticsearch > Remote Clusters* to get started. 
 
 [role="screenshot"]
 image::images/add_remote_cluster.png[][UI for adding a remote cluster]
 
+Once a remote cluster is registered, you can use the tools under *Cross Cluster Replication* 
+to add and manage follower indices on the local cluster, and replicate data from 
+indices on the remote cluster based on an auto-follow index pattern.
+
+[float]
+[[managing-cross-cluster-replication]]
+== Managing cross cluster replication
+ 
+*Cross Cluster Replication* includes tools to help you create and manage the cross cluster 
```

Comment:
Is tools the correct word?

Reply:
Good question, I was trying to avoid the words apps, features, and UIs, since those seem to be problematic for some. Which word do you think works best?

Reply:
How about leaving the word out.  Cross Cluster Replications helps you create ..."

Reply:
that works :)